### PR TITLE
Update changelog with v3.11.0

### PR DESCRIPTION
**Pull request by adambarreiro**
- Updates changelog with changes from v3.11.0
- Bumps Go SDK to v2.22.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,116 @@
...
+* **New Data Source:** `vcd_nsxt_edgegateway_dns` to read Edge Gateway DNS forwarder configuration ([#1154](https://github.com/vmware/terraform-provider-vcd/pull/1154))
+* **New Resource:** `vcd_vm_vgpu_policy` to manage VM vGPU compute policy configuration ([#1167](https://github.com/vmware/terraform-provider-vcd/pull/1167))
+* **New Data Source:** `vcd_vm_vgpu_policy` to read VM vGPU compute policies ([#1167](https://github.com/vmware/terraform-provider-vcd/pull/1167))
+
+### EXPERIMENTAL FEATURES
+* New guide `Importing resources` on how to import resources with new experimental [Terraform import blocks](https://developer.hashicorp.com/terraform/language/import) ([#1104](https://github.com/vmware/terraform-provider-vcd/pull/1104))
+* New example `Importing catalog contents` showing how to import shared catalogs ([#1104](https://github.com/vmware/terraform-provider-vcd/pull/1104))
+* New example `Importing cloned vApps` showing how to import vApps and VMs from cloned vApps ([#1104](https://github.com/vmware/terraform-provider-vcd/pull/1104))
```
Comment (Didainius) [inline CHANGELOG.md:34]: Duplicate of https://github.com/vmware/terraform-provider-vcd/pull/1181/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR6-R8

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,116 @@
...
+* **New Data Source:** `vcd_rde_behavior_invocation` to invoke a Behavior of a given RDE ([#1117](https://github.com/vmware/terraform-provider-vcd/pull/1117), [#1136](https://github.com/vmware/terraform-provider-vcd/pull/1136))
+* **New Resource:** `vcd_vm_vgpu_policy` to manage VM vGPU compute policy configuration ([#1167](https://github.com/vmware/terraform-provider-vcd/pull/1167))
+* **New Data Source:** `vcd_vm_vgpu_policy` to read VM vGPU compute policies ([#1167](https://github.com/vmware/terraform-provider-vcd/pull/1167))
```
Comment (Didainius) [inline CHANGELOG.md:37]: Duplicate of https://github.com/vmware/terraform-provider-vcd/pull/1181/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR28-R29

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,116 @@
...
+
+### EXPERIMENTAL FEATURES
+* New guide `Importing resources` on how to import resources with new experimental [Terraform import blocks](https://developer.hashicorp.com/terraform/language/import) ([#1104](https://github.com/vmware/terraform-provider-vcd/pull/1104))
+* New example `Importing catalog contents` showing how to import shared catalogs ([#1104](https://github.com/vmware/terraform-provider-vcd/pull/1104))
+* New example `Importing cloned vApps` showing how to import vApps and VMs from cloned vApps ([#1104](https://github.com/vmware/terraform-provider-vcd/pull/1104))
+* **New Data Source:** `vcd_rde_behavior_invocation` to invoke a Behavior of a given RDE ([#1117](https://github.com/vmware/terraform-provider-vcd/pull/1117), [#1136](https://github.com/vmware/terraform-provider-vcd/pull/1136))
```
Comment (Didainius) [inline CHANGELOG.md:35]: Duplicate of https://github.com/vmware/terraform-provider-vcd/pull/1181/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR10

File: CHANGELOG.md (hunk 1 above)
Comment (Didainius) [reply CHANGELOG.md:34]: A note, that IMO we should probably remove from `FEATURES` and leave it only in `EXPERIMENTAL FEATURES`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,117 @@
...
+* **New Resource:** `vcd_nsxt_edgegateway_l2_vpn_tunnel` to manage Edge Gateway L2 VPN Tunnel sessions ([#1061](https://github.com/vmware/terraform-provider-vcd/pull/1061))
+* **New Data Source:** `vcd_nsxt_edgegateway_l2_vpn_tunnel` to read Edge Gateway L2 VPN Tunnel sessions ([#1061](https://github.com/vmware/terraform-provider-vcd/pull/1061))
+* **New Resource:** `vcd_nsxt_edgegateway_dns` to manage Edge Gateway DNS forwarder configuration ([#1154](https://github.com/vmware/terraform-provider-vcd/pull/1154))
+* **New Data Source:** `vcd_nsxt_edgegateway_dns` to read Edge Gateway DNS forwarder configuration ([#1154](https://github.com/vmware/terraform-provider-vcd/pull/1154))
+* **New Resource:** `vcd_vm_vgpu_policy` to manage VM vGPU compute policy configuration ([#1167](https://github.com/vmware/terraform-provider-vcd/pull/1167))
+* **New Data Source:** `vcd_vm_vgpu_policy` to read VM vGPU compute policies ([#1167](https://github.com/vmware/terraform-provider-vcd/pull/1167))
```
Comment (lvirbalas) [inline CHANGELOG.md:30]: Shouldn't these be in experimental section? @adezxc

**Review by lvirbalas — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (adambarreiro) [reply CHANGELOG.md]: Yes, it seems they are duplicated. Removed the one entry from FEATURES list.

File: CHANGELOG.md (hunk 3 above)
Comment (adambarreiro) [reply CHANGELOG.md:30]: Good catch, removed

File: CHANGELOG.md (hunk 2 above)
Comment (adambarreiro) [reply CHANGELOG.md:32]: Good catch, removed

File: CHANGELOG.md (hunk 1 above)
Comment (adambarreiro) [reply CHANGELOG.md:29]: Good catch, removed

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,116 @@
...
+## 3.11.0 (December 12, 2023)
 
-Changes in progress for v3.11.0 are available at [.changes/v3.11.0](https://github.com/vmware/terraform-provider-vcd/tree/main/.changes/v3.11.0) until the release.
+### FEATURES
+* Add support for **Container Service Extension v4.1** by updating both the installation and the cluster management
+  guides ([#1063](https://github.com/vmware/terraform-provider-vcd/pull/1063), [#1139](https://github.com/vmware/terraform-provider-vcd/pull/1139))
+* New guide `Importing resources` on how to import resources with new experimental [Terraform import blocks](https://developer.hashicorp.com/terraform/language/import) ([#1104](https://github.com/vmware/terraform-provider-vcd/pull/1104))
+* New example `Importing catalog contents` showing how to import shared catalogs ([#1104](https://github.com/vmware/terraform-provider-vcd/pull/1104))
+* New example `Importing cloned vApps` showing how to import vApps and VMs from cloned vApps ([#1104](https://github.com/vmware/terraform-provider-vcd/pull/1104))
+* * **New Resource:** `vcd_network_pool` to create and manage network pools ([#1115](https://github.com/vmware/terraform-provider-vcd/pull/1115))
```
[the remark this answers is not in the document]
Comment (adambarreiro) [reply CHANGELOG.md]: Good catch, removed

**Review by lvirbalas — APPROVED** [review]
I have double checked for duplicates - it's clear now.

And it looks great. Cheers to a big payload!

**Review by Didainius — APPROVED** [review]

**Review by adezxc — APPROVED** [review]

**Pull request merged**